docs/docs/60-mgmt-and-ops/20-architecture.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Ten-Thousand Foot View
+title: Architecture
```

Comment:
Looks like renaming these files is causing some links to broke, hence the failing CI. Same applies for the security-considerations file.